test/JDBC/input/views/sys-server_principals.sql

```diff
@@ -34,3 +34,37 @@ SELECT name, type, type_desc, default_database_name, default_language_name
 FROM sys.server_principals 
 WHERE name in ('jdbc_user', 'serv_principal_test');
 GO
+
+select name, principal_id, sid, type, type_desc
+from sys.server_principals where name = 'public'
+GO
```

Comment:
Other changes looks good. Can you please check login_token and user_token view reflects this as well or not?

Reply:
+1.
There are tests for user_token, please verify login_token as well.